.github/fabricbot.json

```diff
@@ -904,37 +776,6 @@
         ]
       }
     },
-    {
-      "taskType": "trigger",
-      "capabilityId": "IssueResponder",
-      "subCapability": "IssuesOnlyResponder",
-      "version": "1.0",
-      "config": {
-        "conditions": {
-          "operator": "and",
-          "operands": [
-            {
-              "name": "isAction",
-              "parameters": {
-                "action": "closed"
-              }
-            }
-          ]
-        },
-        "eventType": "issue",
-        "eventNames": [
-          "issues",
-          "project_card"
-        ],
-        "taskName": "Remove closed issues from milestones",
-        "actions": [
-          {
-            "name": "removeMilestone",
-            "parameters": {}
-          }
-        ]
-      }
-    },
```

Comment:
Why remove this?

Reply:
The way that I eventually settled on in other repos was to milestone pull-requests - since those represent the code changes that went into a specific milestone. Issues automatically get closed by GitHub itself (assuming PRs use the keywords), and there's no real way for the bot to know which milestone the issue belongs to. Besides, an issue could be resolved by multiple PRs.
An issue at triage time may be tagged with a milestone, but in my experience that's always speculative and, in reality, an issue may get closed manually (e.g., decided its not worth the effort) or resolved by a change in a completely different milestone. So to avoid any possible confusion this step demilestones closed issues.

Hope this makes sense. And I'm happy to discuss this topic further, may be there's some other angle that I miss.